<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1298</width>
    <height>778</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>3D Model Viewer</string>
  </property>
  <property name="windowIcon">
   <iconset resource="ModelViewer.qrc">
    <normaloff>:/new/prefix1/res/logo.png</normaloff>:/new/prefix1/res/logo.png</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QMdiArea" name="mdiArea">
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="background">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>35</red>
         <green>35</green>
         <blue>35</blue>
        </color>
       </brush>
      </property>
      <property name="activationOrder">
       <enum>QMdiArea::ActivationHistoryOrder</enum>
      </property>
      <property name="viewMode">
       <enum>QMdiArea::SubWindowView</enum>
      </property>
      <property name="documentMode">
       <bool>false</bool>
      </property>
      <property name="tabsClosable">
       <bool>false</bool>
      </property>
      <property name="tabsMovable">
       <bool>false</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1298</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
    <addaction name="actionAbout_Qt"/>
   </widget>
   <widget class="QMenu" name="menuWindows">
    <property name="title">
     <string>Windows</string>
    </property>
    <addaction name="actionCascade"/>
    <addaction name="actionTile"/>
    <addaction name="actionTile_Horizontally"/>
    <addaction name="actionTile_Vertically"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuWindows"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionAbout_Qt">
   <property name="text">
    <string>About Qt</string>
   </property>
  </action>
  <action name="actionNew">
   <property name="text">
    <string>New</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionTile_Horizontally">
   <property name="text">
    <string>Tile Horizontally</string>
   </property>
  </action>
  <action name="actionTile_Vertically">
   <property name="text">
    <string>Tile Vertically</string>
   </property>
  </action>
  <action name="actionCascade">
   <property name="text">
    <string>Cascade</string>
   </property>
  </action>
  <action name="actionTile">
   <property name="text">
    <string>Tile</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="ModelViewer.qrc"/>
 </resources>
 <connections/>
</ui>
